app/facades/rails_engine_facade.rb

```diff
@@ -0,0 +1,23 @@
+class RailsEngineFacade
```

Comment:
I really like this service name and the approach overall-- it's specific to the API we're consuming without being so specific that any API consumption code is repeated in multiple services.